Libraries/LibWeb/WebDriver/ElementReference.cpp

```diff
@@ -530,20 +530,19 @@ String element_rendered_text(DOM::Node& node)
 CSSPixelPoint in_view_center_point(DOM::Element const& element, CSSPixelRect viewport)
 {
     // 1. Let rectangle be the first element of the DOMRect sequence returned by calling getClientRects() on element.
-    auto const* rectangle = element.get_client_rects()->item(0);
-    VERIFY(rectangle);
+    auto rectangle = element.get_client_rects().first();
 
     // 2. Let left be max(0, min(x coordinate, x coordinate + width dimension)).
-    auto left = max(0.0, min(rectangle->x(), rectangle->x() + rectangle->width()));
+    auto left = max(CSSPixels(0), min(rectangle.x(), rectangle.x() + rectangle.width()));
```

Comment:
`rectangle.right()`

Reply:
Hmm, let's keep this one looking like the spec even if `right()` would be more concise :)